src/main/java/net/dv8tion/jda/internal/handle/ChannelUpdateHandler.java

```diff
@@ -103,165 +99,62 @@ protected Long handleInternally(DataObject content)
-                }
-                if (!Objects.equals(oldTopic, topic))
-                {
-                    textChannel.setTopic(topic);
-                    getJDA().handleEvent(
-                            new ChannelUpdateTopicEvent(
-                                    getJDA(), responseNumber,
-                                    textChannel, oldTopic, topic));
-                }
-                if (oldPosition != position)
-                {
-                    textChannel.setPosition(position);
-                    getJDA().handleEvent(
-                            new ChannelUpdatePositionEvent(
-                                    getJDA(), responseNumber,
-                                    textChannel, oldPosition, position));
-                }
-                if (oldNsfw != nsfw)
-                {
-                    textChannel.setNSFW(nsfw);
-                    getJDA().handleEvent(
-                            new ChannelUpdateNSFWEvent(
-                                    getJDA(), responseNumber,
-                                    textChannel, oldNsfw, nsfw));
-                }
-                if (oldSlowmode != slowmode)
-                {
-                    textChannel.setSlowmode(slowmode);
-                    getJDA().handleEvent(
-                            new ChannelUpdateSlowmodeEvent(
-                                    getJDA(), responseNumber,
-                                    textChannel, oldSlowmode, slowmode));
-                }
-                if (oldDefaultThreadSlowmode != defaultThreadSlowmode)
-                {
-                    textChannel.setDefaultThreadSlowmode(defaultThreadSlowmode);
-                    getJDA().handleEvent(
-                            new ChannelUpdateDefaultThreadSlowmodeEvent(
-                                    getJDA(), responseNumber,
-                                    textChannel, oldDefaultThreadSlowmode, defaultThreadSlowmode));
-                }
-                break;
-            }
             case FORUM:
-            {
-                final String topic = content.getString("topic", null);
-                final EmojiUnion defaultReaction = content.optObject("default_reaction_emoji")
+                ForumChannelImpl forumChannel = (ForumChannelImpl) channel;
+                EmojiUnion defaultReaction = content.optObject("default_reaction_emoji")
                         .map(json -> {
                             json.opt("emoji_id").ifPresent(id -> json.put("id", id));
                             json.opt("emoji_name").ifPresent(n -> json.put("name", n));
                             return EntityBuilder.createEmoji(json);
                         })
                         .orElse(null);
 
-                ForumChannelImpl forumChannel = (ForumChannelImpl) channel;
                 content.optArray("available_tags").ifPresent(array -> handleTagsUpdate(forumChannel, array));
+                int flags = content.getInt("flags", 0);
```

Comment:
Lets not have a mutation in between all of the variable initializations.

Lets move this handleTagsUpdate call to be after them.